### PR TITLE
chore: update issue and pr templates

### PR DESCRIPTION
**Pull request by SiTaggart**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      label: "Steps to reproduce"
+      description: |
+        Steps to reproduce the behavior:
+      value: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+        4. See error
+  - type: "input"
+    id: "chakra-version"
```
Comment (shleewhite) [inline .github/ISSUE_TEMPLATE/bug_report.yml:44]: Suggestion: maybe the id should just be 'version', but good to know where you got this form from 🤣

File: .github/PULL_REQUEST_TEMPLATE/contribution_request_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Details of change...
+
+### Change Two
+
+Details of change...
+
+## Checklist
+
+- [ ] Work in a "Draft" branch whilst you are getting feedback to reduce Chromatic costs. Once you are ready for approval, convert the PR to "Ready to review"
+- [ ] Ensure all `required`a checks have passed
```
Comment (shleewhite) [inline .github/PULL_REQUEST_TEMPLATE/contribution_request_template.md:16]:
```suggestion
- [ ] Ensure all `required` checks have passed
```

File: .github/PULL_REQUEST_TEMPLATE/internal_request_template.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Details of change...
+
+### Change Two
+
+Details of change...
+
+## Checklist
+
+- [ ] Work in a "Draft" branch whilst you are getting feedback to reduce Chromatic costs. Once you are ready for approval, convert the PR to "Ready to review"
+- [ ] Ensure all `required`a checks have passed
```
Comment (shleewhite) [inline .github/PULL_REQUEST_TEMPLATE/internal_request_template.md:16]:
```suggestion
- [ ] Ensure all `required` checks have passed
```

**Review by shleewhite — COMMENTED** [review]
Just a couple small things, otherwise looks good!

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (SiTaggart) [reply .github/ISSUE_TEMPLATE/bug_report.yml:44]: 🥷

File: .github/PULL_REQUEST_TEMPLATE/internal_request_template.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Checklist
+
+- [ ] Work in a "Draft" branch whilst you are getting feedback to reduce Chromatic costs. Once you are ready for approval, convert the PR to "Ready to review"
+- [ ] Ensure all `required` checks have passed
+- [ ] Icon changes must have a product design review
+- [ ] Website content changes must have a product design review
+- [ ] At least two engineers have reviewed any code changes
+- [ ] At least one engineering lead has reviewed any core package changes or repository infrastructure
+- [ ] Website visual regression testing is run by adding `🕵🏻‍♀️ Run website visual regression` label
```
[the remark this answers is not in the document]
Comment (SiTaggart) [reply .github/PULL_REQUEST_TEMPLATE/internal_request_template.md:21]: I wish there was, I haven't found it yet though. If you find anything in the docs let me know

**Review by TheSisb — APPROVED** [review]

**Review by shleewhite — APPROVED** [review]

**Pull request merged**